$OpenBSD: patch-lib_Crypto_Hash___init___py,v 1.7 2014/05/11 20:58:23 sthen Exp $

Patch with reduced context as otherwise CVS Id would get expanded.

--- lib/Crypto/Hash/__init__.py.orig	Mon Oct 14 22:38:10 2013
+++ lib/Crypto/Hash/__init__.py	Sun May 11 19:56:54 2014
@@ -46,10 +46,10 @@ The hashing modules here all support the interface des
 
 .. _`PEP 247` : http://www.python.org/dev/peps/pep-0247/
 
-:undocumented: _MD2, _MD4, _RIPEMD160, _SHA224, _SHA256, _SHA384, _SHA512
+:undocumented: _MD4, _RIPEMD160, _SHA224, _SHA256, _SHA384, _SHA512
 """
 
-__all__ = ['HMAC', 'MD2', 'MD4', 'MD5', 'RIPEMD', 'SHA',
+__all__ = ['HMAC', 'MD4', 'MD5', 'RIPEMD', 'SHA',
            'SHA224', 'SHA256', 'SHA384', 'SHA512']
